Aztec/Classes/Public/Extensions/CGRect+Helpers.swift

```diff
@@ -0,0 +1,56 @@
+import CoreGraphics
+
+extension CGRect {
+    /// The position to which a rect is anchored when resized vertically.
+    ///
+    enum VerticalAnchor {
+        case top
+        case middle
+        case bottom
+    }
+
+    /// The position to which a rect is anchored when resized horizontally.
+    ///
+    enum HorizontalAnchor {
+        case left
+        case center
+        case right
+    }
+
+    /// Return a rect resized to a specific size, while anchoring it to one of
+    /// their edges or centers.
+    ///
+    /// By default, changing a CGRect size will resize it anchored to its top
```

Comment:
I was a bit confused by the wording here as it felt like the resize operation is more than just an assignment:

```By default, changing a CGRect size will resize it anchored to its top left corner left corner, as a CGRect is defined by its `origin`.```

I think something like this could describe the situation better:

```By default, CGRect does not support anchoring, so modifying the size will not affect the origin in any way.```